sdk/core/azure-core/test/e2e/azure_core_with_curl_bodyBuffer.cpp

```diff
@@ -72,18 +72,41 @@ int main()
   return 0;
 }
 
-#if defined(AZ_PLATFORM_POSIX)
 void doFileRequest(Context const& context, HttpPipeline& pipeline)
 {
 
   Azure::Core::Http::Url host("https://httpbin.org/put");
   cout << "Creating a Put From File request to" << endl
        << "Host: " << host.GetAbsoluteUrl() << endl;
 
+#if defined(AZ_PLATFORM_POSIX)
   // Open a file that contains: {{"key":"value"}, {"key2":"value2"}, {"key3":"value3"}}
   int fd = open("/home/vivazqu/workspace/a", O_RDONLY);
```

Comment:
Btw, hardcoding a local file path specific to your machine isn't great. We should fix this.